src/Chainweb/Store/Git/Internal.hs

```diff
@@ -245,26 +246,36 @@ instance TreeDb GitStore where
             g (_, T2 _ blob) =
                 GitStoreBlockHeader <$> lockGitStore gs (\gsd -> readHeader' gsd blob)
 
-    leafEntries gs next limit minr maxr = do
-        ls <- fmap (sortOn _blockHeight) . liftIO $ leaves gs
-        counter <- liftIO $ newIORef 0
-        countItems counter . postprocess . P.map GitStoreBlockHeader $ P.each ls
-        total <- liftIO $ readIORef counter
-        pure (int total, Eos True)
-      where
-        postprocess :: Stream (Of (DbEntry GitStore)) IO () -> Stream (Of (DbEntry GitStore)) IO ()
-        postprocess =
-            maybe id filterItems limit
-            . maybe id seekToNext next
-            . maybe id maxItems maxr
-            . maybe id minItems minr
-
-        minItems :: MinRank -> Stream (Of (DbEntry GitStore)) IO () -> Stream (Of (DbEntry GitStore)) IO ()
-        minItems (MinRank (Min n)) =
-            P.dropWhile (\(GitStoreBlockHeader bh) -> int (_blockHeight bh) < n)
-
     insert gs (GitStoreBlockHeader bh) = void $ insertBlock gs bh
 
+    maxEntry gs = GitStoreBlockHeader <$> maximumBy (compare `on` _blockHeight) <$> leaves gs
+
+leafEntries
```

Comment:
It used to be a class method, but this PR removes it from the class. I hesitated deleting the code completely, because @fosskers and @gregorycollins are the maintainers of this code. The module isn't used at all, so we may consider removing it from the code base. 

@fosskers, @gregorycollins maybe you want to move the git-store into a separate package so that the work that went into it isn't lost?